lib/data/brands.yaml

```diff
@@ -248,16 +235,157 @@
   - :length:
     - 16
     :prefixes:
-    - '4011'
+    - '401178'
+    - '401179'
+    - '431274'
     - '438935'
     - '451416'
-    - '4576'
+    - '457393'
+    - '457631'
+    - '457632'
     - '504175'
+    - '506717'
+    - '506718'
+    - '506720'
+    - '506721'
+    - '506724'
+    - '506725'
+    - '506726'
+    - '506727'
+    - '506728'
+    - '506729'
+    - '506730'
+    - '506731'
+    - '506732'
+    - '506733'
+    - '506739'
+    - '506740'
+    - '506741'
+    - '506742'
+    - '506743'
+    - '506744'
+    - '506745'
+    - '506746'
+    - '506747'
+    - '506748'
+    - '506753'
+    - '506774'
+    - '506775'
+    - '506776'
+    - '506777'
+    - '506778'
     - '506699'
-    - '5067'
+    - '506700'
```

Comment:
@lucianosousa , also I want to ask to make one more improvement might be done

in such cases

 '  -  506700'
     - '506701'
     - '506702'
     - '506703'
     - '506704'
     - '506705'
     - '506706'
     - '506707'
     - '506708'
     - '506709'
you can just put 1 string

 '   -  50670'  
that will include all the range 50670(0..9)